packages/clay-css/src/scss/bootstrap/utilities/_text.scss

```diff
@@ -33,12 +33,13 @@
 
 // Weight and italics
 
-.font-weight-light   { font-weight: $font-weight-light !important; }
-.font-weight-lighter { font-weight: $font-weight-lighter !important; }
-.font-weight-normal  { font-weight: $font-weight-normal !important; }
-.font-weight-bold    { font-weight: $font-weight-bold !important; }
-.font-weight-bolder  { font-weight: $font-weight-bolder !important; }
-.font-italic         { font-style: italic !important; }
+.font-weight-light      { font-weight: $font-weight-light !important; }
+.font-weight-lighter    { font-weight: $font-weight-lighter !important; }
+.font-weight-normal     { font-weight: $font-weight-normal !important; }
+.font-weight-semi-bold  { font-weight: $font-weight-semi-bold !important; }
+.font-weight-bold       { font-weight: $font-weight-bold !important; }
+.font-weight-bolder     { font-weight: $font-weight-bolder !important; }
+.font-italic            { font-style: italic !important; }
```

Comment:
Hey @pat270 I realized that it was also missing here, but looking now this is from the bootstrap, I'm not sure if I should add it here too, let me know if I should remove it from here.

Reply:
@matuzalemsteles We don't need to add this here. The variable `$font-weight-semi-bold` doesn't exist in Bootstrap. This will throw a Sass error if importing Bootstrap only.

Reply:
Thanks @pat270!